web/client/components/app/StandardRouter.jsx

```diff
@@ -22,7 +22,8 @@ const Localized = require('../I18N/Localized');
 const assign = require('object-assign');
 
 const Theme = connect((state) => ({
-    theme: state.theme && state.theme.selectedTheme && state.theme.selectedTheme.id
+    theme: state.theme && state.theme.selectedTheme && state.theme.selectedTheme.id,
+    version: state.version && state.version.current
```

Comment:
Maybe it's better to move the connect of version to app.jsx, introducing a version props also in StandardRouter